config/test.json

```diff
@@ -36,11 +36,11 @@
 	"FEATURE_IDENTITY_MANAGEMENT_ENABLED": true,
 	"IDENTITY_MANAGEMENT": {
 		"URI": "http://localhost:8080/",
-		"TENANT": "dBildungscloud",
```

Comment:
Was this on purpose, or just to run the tests locally?

Reply:
During testing, we need administrative access to create new realms (clean test setup locally and with the CI). Hence, I modified the user so that the keycloak master admin is used.